packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedbackForm.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import FieldInput from 'shared/components/FieldInput';
+import { requiredField } from 'shared/components/Validation/rules';
+import { FieldTextArea } from 'shared/components/FieldTextArea';
+import { Flex, Text } from 'design';
+import Toggle from 'teleport/components/Toggle';
+import { ShareFeedbackFormValues } from './types';
+
+interface ShareFeedbackFormProps {
+  formValues: ShareFeedbackFormValues;
+
+  setFormValues(values: ShareFeedbackFormValues): void;
+}
+
+export function ShareFeedbackForm({
+  formValues,
+  setFormValues,
+}: ShareFeedbackFormProps) {
+  const isFieldRequiringEmailEnabled =
+    formValues.newsletterEnabled || formValues.salesContactEnabled;
```

Comment:
Should we just make it to be always required? IIRC that's what we decided on during our recent conversation in DMs and the designs also seem to indicate that the email field is required.

Reply:
I think we didn't decide whether it should be required or not, but let's keep things simple. I got rid of this condition.